build/dockerfiles/rhel.Dockerfile

```diff
@@ -122,16 +125,21 @@ FROM builder AS offline-builder
 # built in Brew, use tarball in lookaside cache; built locally, comment this out
 # COPY v3.tgz /tmp/v3.tgz
 
-# to get all the python deps pre-fetched so we can build in Brew:
+# to get all the cached vsix files pre-fetched so we can use them in Brew:
 # 1. extract files in the container to your local filesystem
 #    CONTAINERNAME="pluginregistryoffline" && docker build -t ${CONTAINERNAME} . --target=offline-builder --no-cache --squash --build-arg BOOTSTRAP=true
 #    mkdir -p /tmp/pr-res/ && docker run -it -v /tmp/pr-res/:/tmp/pr-res/ ${CONTAINERNAME} /bin/bash -c "cd /build/v3/ && cp -r ./* /tmp/pr-res/"
 #    pushd /tmp/pr-res >/dev/null && sudo tar czf v3.tgz ./* && popd >/dev/null && mv -f /tmp/pr-res/v3.tgz . && sudo rm -fr /tmp/pr-res/
 
 # 2. then add it to dist-git so it's part of this repo
 #    rhpkg new-sources root-local.tgz v3.tgz
-RUN if [ ! -f /tmp/v3.tgz ] || [ ${BOOTSTRAP} == "true" ]; then \
-      ./cache_artifacts.sh v3 && chmod -c -R g+rwX /build; \
+RUN if [[ ! -f /tmp/v3.tgz ]] || [[ ${BOOTSTRAP} == "true" ]]; then \
+      # To only cache files from /latest/ folders, use --build-arg LATEST_ONLY=true 
+      if [[ ${LATEST_ONLY} == "true" ]]; then \
+        ./cache_artifacts.sh v3 --latest-only && chmod -R g+rwX /build; \
+      else \
```

Comment:
This branch of the `if` does nothing, since `${LATEST_ONLY} == true` will cause non-latest metas to be deleted in an earlier step (line 76).

You'd need a separate arg to keep all metas and cache only latest.